### ISSUE TITLE
Mejora la previsualisacion del repo

### ISSUE THREAD

**Issue by Revay3d** [anuraghazra/github-readme-stats#2951]
## Idea

Sería útil y bonito que mejoraran la presentación del repositorio para que se viera así o mejor. El logo del autor o imagen personalizada y cambiar colores como los demás e idiomas y bordes redondos sería algo que mejoraría mucho.

## Ejemplo de como puede ser
![RevayDev](https://github.com/anuraghazra/github-readme-stats/assets/99051274/d13d2cd2-33eb-4552-8f25-66ad881e9990)

**Comment by rickstaa** [comment]
Cool idea! Feel free to create a pull request.

> ¡Buena idea! Siéntase libre de crear una solicitud de extracción.
> *translated by [google translate](https://translate.google.com/?hl=nl&sl=en&tl=es&text=Cool%20idea!%20Feel%20free%20to%20create%20a%20pull%20request.&op=translate)*

**Comment by qwerty541** [comment]
Good idea. But I don't like your wording "Improve repo preview". We should keep current version as well. It should be implemented as layout parameter for repo card enpoint. For example: `&layout=pin` and `&layout=opengraph`. We already have same for top languages and wakatime cards.

**Comment by Revay3d** [comment]
hola, gracias por sus consejos los tomare en cuenta 👍🏼

**Comment by Lokesh-Bijarniya** [comment]
Is this issue open still?
